### PR TITLE
Minor: remove duplicated initial delay in KC health check config

### PR DESCRIPTION
**Pull request by fvaleri**
Minor: remove duplicated initial delay in KC health check config.

### CODE REVIEW

File: cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public class KafkaConnectCluster extends AbstractModel implements SupportsMetric
     protected static final String LOG_AND_METRICS_CONFIG_VOLUME_MOUNT = "/opt/kafka/custom-config/";
 
     // Configuration defaults
-    private static final Probe DEFAULT_HEALTHCHECK_OPTIONS = new ProbeBuilder().withInitialDelaySeconds(5).withInitialDelaySeconds(60).build();
+    private static final Probe DEFAULT_HEALTHCHECK_OPTIONS = new ProbeBuilder().withInitialDelaySeconds(60).build();
```
Comment (scholzj) [inline cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java:104]: Was the `5` supposed to be something else?

Comment (scholzj) [reply cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java:104]: If you browse through the history of the file, it seems like the 5 should have been the timeout.

Comment (fvaleri) [reply cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java]: It is actually the same value ad the default Probe value, but it's better to be explicit.

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Pull request merged**